index.js

```diff
@@ -100,6 +109,13 @@ exports.methodLoader = async function(server, options, useAsPlugin) {
           }
           if (method.method) {
             server.method(key, method.method, method.options);
+            const registeredMethod = _.get(server.methods, key);
+            if (method.description) {
```

Comment:
what is this for?

Reply:
Don't we need to set this info on the registered server method, so that it is available for hapi-docs?  If I take this out the new test fails.

Reply:
What is the change above then?